src/main/resources/logstash-rest.yml

```diff
@@ -38,3 +38,7 @@ logstash_nodes_hot_threads_human:
 logstash_plugins:
   versions:
     "> 0.0.0": "/_node/plugins"
+
+logstash_health_report:
```

Comment:
Can we sort this so it's in a predictable spot?

Reply:
Should I sort the entire list of APIs?

Reply:
Ah, no. I hadn't realized Logstash had not been sorted properly. I'll merge this, then do it all.